### PR TITLE
Add an option to display issue comments

### PR DESCRIPTION
**Pull request by shulard**
The function used to retrieve the list of issue comments was wrong because it retrieve the issue list instead of the comment list...

Example of output :

```
Issue #2 (invalid): by xxx [xxx]
Type: Issue
Milestone: v0.0.1
Labels: enhancement, major, Bots activity, v0.0.1
Title: [issue title here]
Link: [issue url here]

**Objective:** To know the pace of crawl engines, and have a state of daily crawl.

Comments -------------------------------------------------------------------------------------------

Comment #14179448 by xxx on Tue, 09 Dec 2014 18:27:17 +0000
Link: [comment url here]

[comment body here]
----------
Comment #14153534 by xxx on Mon, 08 Dec 2014 19:46:18 +0000
Link: [comment url here]

[comment body here]
----------
```

### CODE REVIEW

File: src/Command/Issue/IssueShowCommand.php [hunk 1]
```diff
@@ -83,10 +88,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 'Title: '.$issue['title'],
                 'Link: '.$issue['url'],
                 '',
-                $issue['body'],
+                wordwrap($issue['body'], 100),
             ]
         );
 
+        if (true === $input->getOption('with-comments') && count($comments) > 0) {
```
Comment (pierredup) [inline src/Command/Issue/IssueShowCommand.php]: The variable '$comments' won't always exist here. Variables shouldn't be declared inside conditionals

**Comment by pierredup** [comment]
I'm not sure if displaying all the comments for an issue is a good idea. What about issues with hundreds of comments, it can become difficult to see all the comments.
I rather think we should add an 'issue:comments' command, where you can limit the number of issues returned, add filters (E.G only return comments from a specific user) etc.

**Comment by shulard** [comment]
Hum... Maybe you are right. I though that the option is a nice way to handle a simple approach for comment displaying.
In fact there isn't a way to filter / search in the comments through thirdparty APIs. But we can offer it through PHP.

File: src/Command/Issue/IssueShowCommand.php (hunk 1 above)
Comment (shulard) [inline src/Command/Issue/IssueShowCommand.php]: `$comments` is always defined when the option `with-comments` is true. I understand that you want a cleaner approach here :smile:

**Comment by cordoval** [comment]
is this ready @shulard ?

**Comment by shulard** [comment]
I think yes... I've just rebased today and fix a little issue about comment URL definition for Gitlab adapter.

@pierredup told me that maybe it's not a good idea... You want to merge that one then open a discussion about a new `issue:list:comments` command ?

**Comment by cordoval** [comment]
we can filter with another PR. But this is a step forward.

**Pull request merged**